references/sdk.md

```diff
@@ -67,6 +78,13 @@ Create many data sample assets.
     "test_only": bool,
 }
 ```
+Create multiple data samples through a single HTTP request.
+
+For the `local` argument, please refer to the method `Client.add_data_sample`.
```

Comment:
It's not exactly the same though, the structure changes for data_samples (plural): the path must point to a directory containing only subdirectories, each of which will be turned into a data sample.

Reply:
Yes, your comment applies only to the `path` argument. Is that correct?
I think the `local` argument has exactly the same meaning when adding one or many data samples.

Reply:
Absolutely, sorry. You're right, local has the same meaning it's path that changes.

Reply:
I added some additional info for the `paths` field.
It's only in the CLI that the `path` option can point to a directory containing subdirectories with the option `multiple` set to True.